client/packages/requisitions/src/ResponseRequisition/DetailView/Toolbar.tsx

```diff
@@ -70,11 +80,53 @@ export const Toolbar: FC = () => {
                   />
                 }
               />
-              {showInfo && (
-                <InfoPanel message={t('info.no-shipment')} />
+              {orderType && (
+                <InputWithLabelRow
+                  label={t('label.order-type')}
+                  Input={
+                    <BasicTextInput
+                      disabled={true}
+                      size="small"
+                      sx={{ width: 250 }}
+                      value={orderType ?? ''}
+                    />
+                  }
+                />
+              )}
+              {programName && (
+                <InputWithLabelRow
+                  label={t('label.program')}
+                  Input={
+                    <BasicTextInput
+                      disabled={true}
+                      size="small"
+                      sx={{ width: 250 }}
+                      value={programName ?? ''}
+                    />
+                  }
+                />
+              )}
+              {period && (
+                <InputWithLabelRow
+                  label={t('label.period')}
+                  Input={
+                    <BasicTextInput
+                      disabled={true}
+                      size="small"
+                      sx={{ width: 250 }}
+                      value={period?.name ?? ''}
+                    />
+                  }
+                />
               )}
             </Box>
           </Box>
+          <Box display="flex" flexDirection="row" gap={4}>
+            <Box display="flex" flex={1} flexDirection="column" gap={1}></Box>
+            <Box display="flex" flex={4} flexDirection="column" gap={6}>
+              {showInfo && <InfoPanel message={t('info.no-shipment')} />}
+            </Box>
+          </Box>
```

Comment:
The panel is squashed up - and I don't see the need. How does this look?
Also, no need for the whole box bit if the `showInfo` is false, so I lifted that up
```suggestion
          {showInfo && (
            <Box padding={2}>
              <InfoPanel message={t('info.no-shipment')} />
            </Box>
          )}
```

was:
<img width="985" alt="Screenshot 2023-04-20 at 8 59 11 AM" src="https://user-images.githubusercontent.com/9192912/233199289-c1068cc2-276f-49fc-b1dd-52883af6a7e5.png">


now:

<img width="908" alt="Screenshot 2023-04-20 at 9 02 11 AM" src="https://user-images.githubusercontent.com/9192912/233199307-45e4c1ed-b701-48b6-9906-40c772c0ad87.png">